components/ListView/BlueprintContents.js

```diff
@@ -41,19 +41,28 @@ const BlueprintContents = props => {
     filterValues,
     errorState,
     fetchingState,
-    fetchDetails
+    fetchDetails,
+    pastLength,
+    undo
   } = props;
 
   const { formatMessage } = props.intl;
 
   return (
     <div>
       {(fetchingState === true && <Loading />) ||
-        ((Object.keys(errorState).length && (
+        ((Object.keys(errorState).length > 0 && (
           <EmptyState
             title={formatMessage(messages.emptyStateErrorTitle)}
             message={formatMessage(messages.emptyStateErrorMessage)}
-          />
+          >
+            <p>{errorState.msg}</p>
```

Comment:
The `EmpytState` component has children prop with an React element type. But here's an array, not an element. Here're two solutions:
1. put `<p>` and `<button>` into a `<div>`
2. update children type to:
```javascript
children: PropTypes.oneOfType([
  PropTypes.element,
  PropTypes.arrayOf(PropTypes.element)
])
```
![empty_state](https://user-images.githubusercontent.com/672230/52171938-c717ff00-27a0-11e9-9e42-2a43528efe9d.png)

Reply:
Thanks for the suggestion! I made this update.